cmd/utils_test.go

```diff
@@ -486,3 +487,32 @@ func TestGetMinioMode(t *testing.T) {
 	testMinioMode(globalMinioModeGatewayPrefix + globalGatewayName)
 
 }
+
+func TestTimedValue(t *testing.T) {
+	var cache timedValue
+
```

Comment:
Since you have long wait, add Parallel, so it can run alongside others with this.
```suggestion
	t.Parallel()
```